bin/coverage

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
```

Comment:
I wonder if it's a good idea to use this file for these small packages... all the complication is due to the way we run retro-fitted tests in Desk, and can be completely replaced with a single line in most cases.  Even on larger internal projects, I expect we'll likely never need Desk's complication again, as long as we start the projects with tests.